# refers to the {T} frame cartesian states
std_msgs/Header header
float32 p_e_s_x 
float32 p_e_s_y
float32 p_e_s_z
float32 v_e_s_x 
float32 v_e_s_y
float32 v_e_s_z